fast/stages/FAQ.md

```diff
@@ -15,9 +15,14 @@
         <img src="IaC_SA.png" alt="CICD SA and IaC SA">
     </p>
 
-    
-
-
+## Authenciation 
```

Comment:
Typo :) Can you also lint Markdown? In VSCode there is an extension for that. A newline is needed after headings.

Reply:
Fixed Typo and using lint in Markdown now.